Assets/GothicVR/Scripts/Phoenix/Util/CheckIfAudible.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace GVR.Phoenix.Util
+{
+    public class CheckIfAudible : MonoBehaviour
+    {
+        AudioListener audioListener;
+        AudioSource audioSource;
+        float distanceFromPlayer;
+        void Start()
+        {
+            // Finds the Audio Listener and the Audio Source on the object
+            audioListener = Camera.main.GetComponent<AudioListener>();
+            audioSource = gameObject.GetComponent<AudioSource>();
+        }
+        void Update()
```

Comment:
Ok. I see your point. I'm just curious if this will cause performance issues when we do checks for every sound in the world every frame. But maybe this is nothing to be solved here...
My ideas for the future (just FYI):
1. We spawn world objects only if we reached a certain area (like when in OC, we have OC vobs only and the rest is disabled/removed)
2. You could also set a Coroutine with a condition WaitUntil() then it's only triggered if condion is met. (But in the end the mathematics will be the same. Just at another spot 🤔 )

Do you see a problem here?

Reply:
I don't believe the current approach is that computationally expensive. We could profile this implementation to see if it affects the performance on Quest and Pico, but from my testing it doesnt seem to be an issue